### PR TITLE
added instructions to download gradle on windows10 and fix build failure when running gradle clean dist

### PR DESCRIPTION
**Pull request by Ezmerald**
When I was running gradle clean dist on my windows10 as per Fineract-platform Installation on Windows tutorial, I first got gradle command not found then after I installed gridle I got a build failure after running again gradle clean dist. I fixed the issue and added the solution steps to README.md file

Error fixed:
FAILURE: Build failed with an exception.* What went wrong:Could not
> resolve all dependencies for configuration ':compile'.> Could not resolve
> pentaho-reporting-engine:pentaho-reporting-engine-classic-core:3.9.1.1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,18 @@ Run the following commands:
...
+go to https://gradle.org/
+click "install gradle" (https://gradle.org/install/)
+click "installing manually" (https://gradle.org/install/#manually)
+click "releases page" (https://gradle.org/releases/)
+go to version 2.10
+click complete;
+after the zip folder downloads extract the files into your C folder ( directory: "C:\gradle-2.10" );
+go to control panel -> system and security -> system -> advanced system settings -> environment variables; 
+under user variables click path -> edit -> new -> C:\gradle-2.10\bin then 
+click OK
```
Comment (xurror) [inline README.md:54]: I don't think this is necessary. Fineract platform ships with gradlew.bat by default which is which is an alternative to the standard gradlew for unix systems.
Running the gradle batch file takes care of installing gradle for you automatically.

**Review by xurror — COMMENTED** [review]
Alternatively you can try:
cd fineract
gradlew.bat tomcatRunWAR
Can you try that and see if it works.

**Comment by xurror** [comment]
Also I don't understand why you have 4 commits and 5 edited files. Ithought you were trying to update the readme alone.
Do you mind checking that??

File: README.md (hunk 1 above)
Comment (Ezmerald) [reply README.md:54]: @xurror I was following the tutorial on this link https://cwiki.apache.org/confluence/display/FINERACT/Fineract-platform+Installation+on+Windows 
and it did not include the step of running the gradlew.bat file. I removed the path I created for gradle in my user environment and ran gradlew.bat file then went to gitbash and ran gradle clean dist and it built successfully so I guess what you suggested above works. 
 I have imagebot that I installed as part of the github education pack. It gets installed automatically in my repos which explains commits 
9e78318 and  3751ebc and I am sorry these commits are not relevant. Should probably deactivate imagebot :) I was only trying to edit README.md which I did in commit 
507e463 and then I added semi columns to make the instructions clearer in commit 4a1f521

Comment (xurror) [reply README.md:54]: Right, so on windows you got to use gradlew.bat

I guess there is no need for this PR anymore.
So can you just close this?

**Comment by xurror** [comment]
Thanks for pointing this out.
Hope to see more contributions from you.

**Review by vorburger — CHANGES_REQUESTED** [review]
As @xurror said, this should not be needed.  @Ezmerald we would love you to update the Wiki! Will you?

**Comment by Ezmerald** [comment]
@vorburger sure would love to! I will go ahead and close this pull request and get to updating the wiki in the next couple of days. I have joined the confluence platform but can't see the edit button for the wiki page. Mind pointing me to how I can edit the page? Thanks.

**Comment by vorburger** [comment]
> I have joined the confluence platform but can't see the edit button for the wiki page. Mind pointing me to how I can edit the page?

@Ezmerald we need to grant you Edit permission. I just tried, but cannot find a user named "Ezmerald" on https://cwiki.apache.org/confluence/ to let you edit https://cwiki.apache.org/confluence/display/FINERACT/Fineract-platform+Installation+on+Windows ... under what name did you register?

**Comment by Ezmerald** [comment]
@vorburger the username I used is mhammoud.sara

**Comment by vorburger** [comment]
> @vorburger the username I used is mhammoud.sara

@Ezmerald retry it now, I've granted you some permission which hopefully lets you edit the Wiki.

**Comment by Ezmerald** [comment]
@vorburger I checked and think I can edit the pictures. I have exams this week so will look into it this weekend/early next week and let you know what I did or if I face any problems. Wish me luck on my exams :)

**Comment by Ezmerald** [comment]
@vorburger when I reopened the fineract folder in visual studio and  Git Bash to run the commands, I did as we discussed above and ran the gradlew.bat file then cd into fineract-provider and ran "gradle clean dist" and got "gradle: command not found". I had to go into the variables section of my environment section and add path "C:\gradle-2.10\bin". When I ran "gradle clean dist" again it built successfully. I am assuming that last time when I ran gradlew.bat the built was successful because the project was already open in my laptop and I executed the command after I had a successful build with the changes I had made.

**Comment by vorburger** [comment]
gradle != ./gradlew

>

**Pull request closed without merging**